python/pyspark/sql/connect/expressions.py

```diff
@@ -489,7 +489,20 @@ def __repr__(self) -> str:
             ts = TimestampNTZType().fromInternal(self._value)
             if ts is not None and isinstance(ts, datetime.datetime):
                 return ts.strftime("%Y-%m-%d %H:%M:%S.%f")
-        # TODO(SPARK-49693): Refine the string representation of timedelta
+        elif isinstance(self._dataType, DayTimeIntervalType):
+            delta = DayTimeIntervalType().fromInternal(self._value)
+            if delta is not None and isinstance(delta, datetime.timedelta):
+                try:
+                    import pandas as pd
```

Comment:
spark connect requires pyarrow/pandas iirc so you won't need to handle exceptions here if that's what you're covering

Reply:
yeah, the `pandas` is a mandatory dependency for connect, let me remove this `try-catch`